definitions/ext-sap_idoc/golden_metrics.yml

```diff
@@ -0,0 +1,24 @@
+avgProcTime:
+  title: Avg Processing Time
+  unit: SECONDS
+  query:
+    select: average(NR.SAP.IDOC.PROCESS_TIME)
+    from: Metric
+    where: metricName = `NR.SAP.IDOC.PROCESS_TIME`
```

Comment:
Is this where needed if the select is for a dimensional metric already?

Reply:
Yes. The where is not needed. Will make the change.

Reply:
Updated, where clause removed